webapp/components/channel_switch_modal.jsx

```diff
@@ -27,14 +27,32 @@ export default class SwitchChannelModal extends React.Component {
         this.onExited = this.onExited.bind(this);
         this.handleKeyDown = this.handleKeyDown.bind(this);
         this.handleSubmit = this.handleSubmit.bind(this);
+        this.handleDmUserChange = this.handleDmUserChange.bind(this);
         this.suggestionProviders = [new SwitchChannelProvider()];
 
         this.state = {
+            dmUsers: UserStore.getDirectProfiles(),
+            dmUsersLoaded: false,
```

Comment:
Is this ever used anywhere? We also don't appear to call out to load these if they haven't already been loaded.